.github/workflows/ci.yml

```diff
@@ -35,6 +31,8 @@ jobs:
 
       - name: Release snapshot
         run: |
+          echo "$( jq '.version = "0.0.0"' package.json )" > package.json
+          echo -e "---\n'stylelint-config-primer': patch\n---\n\nFake entry to force publishing" > .changeset/force-snapshot-release.md
```

Comment:
This seems dangerous to run on `master`, right?

Reply:
This is a really cool solution for feature branches, but I'm having a hard time reasoning about how this will work on the main branch.

Reply:
Maybe I'm missing something but wouldn't it work the same way as feature branches? Each commit to any branch (including the main branch) will get a `0.0.0-...` prerelease published to npm. Additionally, PRs merged into the main branch from the changeset bot will trigger an actual release.

In any case, I'd be fine with not publishing prereleases from the main branch, but it'd be nice if tests still ran. Maybe we could separate the two; tests always run, the publish only runs on non-main branches (even if tests eventually fail from a separate workflow)

Reply:
> wouldn't it work the same way as feature branches

Specifically, these file changes don't get committed/pushed anywhere

Reply:
ahh you're right, this wouldn't affect the `release` workflow at all. I'm fine with this 👍